interfaces/api/v1/core/export.py

```
@@ -0,0 +1,104 @@
+        # 对文件名进行URL编码，避免中文编码错误
+        encoded_filename = urllib.parse.quote(filename)
+        
+        # 返回流式响应
+        return StreamingResponse(
+            iter([file_content]),
+            media_type=media_type,
+            headers={
+                "Content-Disposition": f"attachment; filename={encoded_filename}; filename*=UTF-8''{encoded_filename}"
+            }
+        )
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)